docs/next/guides/integrate-with-angular/angular-modules.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Using modules with Angular
+metaTitle: Using modules with Angular - Guide - Handsontable Documentation
+permalink: /next/angular-modules
+canonicalUrl: /angular-modules
+---
+
+# Using modules with Angular
+
+[[toc]]
+
+## Overview
+
+To reduce the size of your app, you can use Handsontable by importing individual [modules](@/guides/building-and-testing/modules.md).
+
+## Using modules with Angular
+
+To use modules with Handsontable's [Angular wrapper](@/guides/integrate-with-angular/angular-installation.md), follow the steps below:
+
+### Step 1: Import the `Base` module
+In the entry point file of your application, import Handsontable's `Base` module:
+  ```js
+  import { BrowserModule } from '@angular/platform-browser';
+  import { NgModule } from '@angular/core';
+  import { AppComponent } from './app.component';
+  import { HotTableModule } from '@handsontable/angular';
+
+  // import Handsontable's `Base` module
+  import Handsontable from 'handsontable/base';
+  ```
+
+### Step 2: Import modules and their registering functions
+Import the required modules (for the list of Handsontable modules, see the [modules cheatsheet](@/guides/building-and-testing/modules.md#modules-cheatsheet)).
```

Comment:
WDYT to rephrase the sentence "Import the required modules..." to something like "Import the modules that you want to use..."? "required" sounds to "required" for me 🤔

Reply:
You're right, "required" is ambiguous here. Done: 7841591